ichk/command.py

```diff
@@ -56,8 +56,12 @@ def entry():
 
     session.connection_timeout = args.timeout
 
-    run(session, args)
-
+    try:
+        run(session, args)
+    except:
+        raise
+    finally:
+        session.cleanup()
```

Comment:
Consider:
```python
with session:
    run(session, args)
```
Should have the same effect as the proposed change (iRODSSession cleans up on `__exit__`).
Rest LGTM.

Reply:
Thanks, you're right. I've improved it.